core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java

```diff
@@ -32,17 +32,34 @@
 
 package org.mskcc.cbio.portal.dao;
 
-import org.mskcc.cbio.portal.model.*;
-import org.mskcc.cbio.portal.util.MutationKeywordUtils;
-
-import org.apache.commons.lang.StringUtils;
-
-import java.sql.*;
-import java.util.*;
-import java.util.regex.*;
 import org.apache.commons.collections.MapIterator;
 import org.apache.commons.collections.keyvalue.MultiKey;
 import org.apache.commons.collections.map.MultiKeyMap;
+import org.apache.commons.lang.StringUtils;
+import org.mskcc.cbio.portal.model.CancerStudy;
+import org.mskcc.cbio.portal.model.CanonicalGene;
+import org.mskcc.cbio.portal.model.ClinicalAttribute;
+import org.mskcc.cbio.portal.model.ExtendedMutation;
+import org.mskcc.cbio.portal.model.GeneticAlterationType;
+import org.mskcc.cbio.portal.model.GeneticProfile;
+import org.mskcc.cbio.portal.model.Sample;
+import org.mskcc.cbio.portal.util.MutationKeywordUtils;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```

Comment:
not sure if people prefer using `*` imports, CC @Luke-Sikina ?

Reply:
fine by me to change like this tho, just not sure if we have a preferred way of import statements

Reply:
I just had feedback by @sheridancbio on another PR where this subject came up. He expressed the desire to not use * for cbioportal packages, but allow it for others. I just turned off the use of wildcards in my IDE all together. Perhaps we can come up with a clear line of code conventions that we can enforce in an automated way?

Reply:
yeah that would be nice, maybe soundcloud has some linting options